src/App.css

```diff
@@ -1,38 +1,508 @@
+  align-items: center;
+}
+.errorMessage .mesageContainer {
+  width: 800px;
+  height: 400px;
+  background-image: url(../public/error-bg.jpg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  display: flex;
+  align-items: center;
+  border: 2px solid black;
+}
+.mesageContainer .textContainer {
+  display: flex;
+  flex-direction: column;
+  height: 50%;
+  justify-content: space-evenly ;
+  padding: 0 50px;
+  background-color: rgba(134, 108, 90, 0.747);
+}
+.mesageContainer .textContainer h2 {
+  font-family: 'Dancing Script', cursive;
+  font-weight: 700;
+  font-size: 3rem;
+  padding-bottom: 1.5rem;
+}
+.mesageContainer .textContainer p {
+  padding: 0 50px;
+}
+
+/* footer */
+.footer {
+  text-align: center;
+  padding: 0 5%;
+  position: fixed;
+  bottom: 0;
+  width: 100vw;
+}
+.footer p {
+  font-size: 0.7rem;
+}
+.footer a {
+  font-size: 0.7rem;
+  color: black;
+}
+.footer a:hover,
+.footer a:focus {
+  color: rgb(112, 130, 62);
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
+
+
+
+/* media queries */
+@media (max-width: 1100px) {
+  /* header */
+  .header {
+    width: auto;
+    position: relative;
+    background-image: url(../public/header-background.jpg);
+    background-repeat: no-repeat;
+    background-position: top center;
+    background-size: cover;
+  }
+  .header .title {
+  height: auto;
+  }
+
+  /* form */
+  .search{
+    margin-bottom: 2rem;
+  }
+  .filter p{
+    cursor: pointer;
+  }
+  .checkBoxes{
+    display: flex;
+    flex-wrap: wrap;
+  }
+  .checkBoxes .options {
+    width: calc(100%/3);
+  }
+
+  /* ApiCall.js */
+  .main {
+    padding-left: 0;
+    min-height: auto;
+  }
+
+  /* GalleryItems.js */
+  .galleryLi {
+    width: 20%;
+    height: 250px;
+  }
+}
+
+
+@media (max-width: 850px) {
+  .checkBoxes .options {
+    width: 50%;
+  }
+  .galleryLi {
+    width: 25%;
+    /* height: 250px; */
+  }
+}
+
+@media (max-width: 690px) {
+  /* GalleryItem.js */
+  p , a {
+    font-size: 0.7rem;
+  }
+  /* Recipe.js */
+  .recipe{
+    border: 5px solid green;
+    grid-template-areas: "b h h h h y"
+                         "f f f f f f"
+                         "c c c m m m"
+                         "t t t d d d"
+                         "p p p p p p"
+                         "p p p p p p"
+                         "l l l s s s"
+                         "i i i i i i";
+  }
+  .recipe .save {
+    text-align: center; 
+}
+  .recipe .ingredientsContainer {
+    margin: 0 auto;
+}
+}
+
+@media (max-width: 650px){
+  h2 {
+    font-size: 1.5rem;
+  }
+}
+
+@media (max-width: 600px) {
+  /* Forms.js */
+  .checkBoxes .options {
+    width: 100%;
+  }
+}
+
+@media (max-width: 520px) {
+  .apiCall .toggle {
+  margin-right: 0.3rem;
+  }
+  h3 {
+    font-size: 1rem;
+  }
+  .galleryLi {
+    width: calc(100%/3);
+    height: 200px;
+  }
+  /* Error.js */
+  .mesageContainer .textContainer {
+  height: 55%;
```

Comment:
Great work on the css and the responsiveness of the app. You also added a setup snippet.